src/System.Private.CoreLib/shared/System/GCMemoryInfo.cs

```diff
@@ -53,4 +53,42 @@ namespace System
             FragmentedBytes = fragmentedBytes;
         }
     }
+
+    // TODO: own file
+    public readonly struct GCConfigInfo
+    {
+        public bool AllowVeryLargeObjects { get; }
+        public bool CpuGroup { get; }
```

Comment:
Many of these look very specialized and platform specific., e.g. `CpuGroup` is a Windows-specific, `LargePages` can have multiple values on Unix, ... . They do not feel like good durable APIs.

Is the purpose of this API to return the config for diagnostic purposes? I am wondering whether it would better to have an API to return all overriden runtime config settings as Dictionary or something. It may be useful to have API proposal issue in CoreFX to discuss.